packages/core/src/components/menu/_menu.scss

```diff
@@ -24,16 +24,16 @@ JavaScript API
 The `Menu`, `MenuItem`, and `MenuDivider` components are available in the __@blueprintjs/core__
 package. Make sure to review the [general usage docs for JS components](#components.usage).
 
-The Menu API includes three stateless React components:
+The menu API includes three stateless React components:
```

Comment:
i think this should remain capitalized... we're talking about our Menu API as a proper noun.